lab_1_classify_by_unigrams/main.py

```diff
@@ -64,6 +85,18 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not (isinstance(predicted, list) and
+            isinstance(actual, list) and
+            len(predicted) == len(actual)):
+        return None
+
+    n = len(predicted)
```

Comment:
rename the variables, it is not clear what they contain

Reply:
Done